### DISCUSSION TITLE
Cannot Apply from France

### DISCUSSION THREAD

**Discussion by faenuccio** [community/community#136953] [GitHub Education]
### Hi everyone,

I am an Associate Professor in a French university and there is no way I can get to use GitHub Education. That's a pity. I tried in all ways to upload a picture of my Card, and it never gets accepted; there is no way to upload a pdf (so, I should jump back to the 70's, print my contract out and try to take a picture). Moreover the semester starts in 2 weeks I am not in France right now. Both these issues should be easy to deal for a "tech company", but I will simply let it go. Bad news for my students...

**Comment by samus-aran** [comment]
Hi @faenuccio 👋 

> my Card, and it never gets accepted;

Can you explain a little more on what you mean by "get accepted". Do you mean that there is a problem with uploading the photo or you are receiving rejection reasons?

**Reply by faenuccio** [reply]
I receive some rejection saying that the system cannot validate it as a proof of my current employment at my University.

**Reply by samus-aran** [reply]
Yeah that usually happens (if there was no other obvious answer, like being far from campus) if the text in the photo was not legible and the application couldn't identify the key elements on the evidence. I suspect that is what has happened. It appears you have been successfully approved with a better quality of photo in your last submission so you should receive your benefits usually within 72 hours of this approval. 👍
